bokehjs/src/lib/models/widgets/tables/row_aggregators.ts

```diff
@@ -0,0 +1,96 @@
+const { Aggregators: { Avg, Min, Max, Sum } } = require('slickgrid/slick.dataview')
```

Comment:
Let's make sure that any new code aligns stylistically with the rest of bokehjs. In this case, no whitespace after `{` and before `}`. This applies to imports as well as plain objects.